docs/docs/features/tls.md

```diff
@@ -1,12 +0,0 @@
-# TLS in BlazingMQ
-
-BlazingMQ supports authenticating brokers using TLS.
-
-
-## Generating Test Certs
-
-```sh
-mkdir -p certs && cd certs
-../bin/gen-tls-certs.sh ca ca-cert blazingmq
-../bin/gen-tls-certs.sh server ca-cert broker_bmqc00_ bmqc00
```

Comment:
Should go to https://github.com/bloomberg/blazingmq/tree/gh-pages
Also let's rename to `bmqtest`